recipes/swagger-spec-validator/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: swagger-spec-validator
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/Yelp/swagger_spec_validator/archive/v{{ version }}.tar.gz
+  sha256: bf3a816e3f44edca2e30ed3af68826e84037ed69d2e985406ba3a5a9e0c0b3a0
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - jsonschema
+    - six
+    - setuptools
+
+test:
+  imports:
+    - swagger_spec_validator
+
+about:
+  home: http://github.com/Yelp/swagger_spec_validator
+  license: Apache-2.0
+  summary: 'Validation of Swagger specifications'
+  license_family: APACHE
+  license_file: LICENSE.txt
+  dev_url: http://swagger-spec-validator.readthedocs.io/
```

Comment:
This should be the `doc_url`, and then add the same url that you have for `home` as the `dev_url`.